src/test/java/org/ays/payload/ApplicationRegistration.java

```diff
@@ -10,4 +14,30 @@ public class ApplicationRegistration {
     private String institutionId;
     private String reason;
 
+    public static String generateApplicationID() {
+        ApplicationRegistration applicationRegistration = new ApplicationRegistration();
+        applicationRegistration.setReason("A valid test string must have forty character.");
+        applicationRegistration.setInstitutionId(AysConfigurationProperty.InstitutionOne.ID);
+        while (true) {
+            Response response = InstitutionEndpoints.postRegistrationAdminApplication(applicationRegistration);
+            String applicationID = response.jsonPath().getString("response.id");
+            if (applicationID != null) {
+                return applicationID;
+            }
+        }
+    }
```

Comment:
Bu fonksiyonu buradan kaldırıp `InstitutionEndpoints`'in içerisine taşıyabiliriz. Burası Application ID üretmiyor, yeniden bir application üretiyor.

Reply:
bu kısmı tam anlayamadım `generateApplicationID` methodunu mu` InstitutionEndpoints` içerisine taşımalıyım?

Reply:
Burayı konuşalım tekrar